README.md

```diff
@@ -73,6 +73,53 @@ The AET dictionary JSON can also be specified directly via the "--aet_dictionary
 
 For the list of command line flags, see [here](import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java)
 
+## Routing to multiple DICOM stores
+
+Import adapter C-STORE service supports routing to multiple DICOM stores based on tags in incoming instance and client AET.
+Filter is in the format fo a QidoRS query, with addition of AETitle virtual tag for matching calling client AET.
```

Comment:
"fo" should be "of"

Reply:
I think calling it a virtual tag might confuse some readers. How about:
"Filter is in the format of a QidoRS query, with the additional option of specifying the AETitle for matching against the calling client AET."

Reply:
Ok